ch22.html

```diff
@@ -252,7 +252,7 @@ <h2>ZonedDateTime class</h2>
             <p><code class="java hljs"><span class="hljs-comment">// Prints 2014-09-19T00:30Z<br /></span>System.out.println(<br />
 	  &nbsp; &nbsp; ZonedDateTime.of(<span class="hljs-number">2014</span>,<span class="hljs-number">9</span>,<span class="hljs-number">19</span>,<span class="hljs-number">0</span>,<span class="hljs-number">30</span>,<span class="hljs-number">0</span>,<span class="hljs-number">0</span>,ZoneId.of(<span class="hljs-string">"Z"</span>)));<br />
 	  <span class="hljs-comment">// Prints 2015-08-31T12:39:27.492-04:00[America/Montreal]<br /></span>System.out.println(<br />
-	  &nbsp; &nbsp; ZonedDateTime.now(ZoneId.of(<span class="hljs-string">"America/Montreal"</span>)));</code></p>
+	  &nbsp; &nbsp; ZonedDateTime.now(ZoneId.of(<span class="hljs-string">"America/Montreal"</span>));</code></p>
```

Comment:
Hi @txpokey, maybe you missed my comment, but I was wondering about this change, I think the third parenthesis is needed (belongs to System.out.println).